src/components/molecules/OakModal/OakModal.test.tsx

```diff
@@ -8,6 +8,9 @@ import { OakModal } from "./OakModal";
 import renderWithTheme from "@/test-helpers/renderWithTheme";
 import { oakDefaultTheme } from "@/styles";
 import { OakThemeProvider } from "@/components/atoms";
+import { installMockIntersectionObserver } from "@/test-helpers";
+
+installMockIntersectionObserver();
```

Comment:
I was getting fed up writing and installing these little mocks into JSDOM so I moved them into our helpers. These might be useful to export from the library at some point.